parler/appsettings.py

```diff
@@ -17,7 +17,7 @@
 # Have to fill the default section explicitly to avoid circular imports
 PARLER_LANGUAGES.setdefault('default', {})
 PARLER_LANGUAGES['default'].setdefault('code', PARLER_DEFAULT_LANGUAGE_CODE)
-PARLER_LANGUAGES['default'].setdefault('fallback', PARLER_DEFAULT_LANGUAGE_CODE)
+PARLER_LANGUAGES['default'].setdefault('fallbacks', [PARLER_DEFAULT_LANGUAGE_CODE])
```

Comment:
What would you like to see here? Should we raise a Deprecation Warning? Or just print something in the console?